### PR TITLE
Test save logs to artifacts dir

### PR DESCRIPTION
**Pull request by rajivnathan**

### CODE REVIEW

File: make/run-cicd-script.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=rajivnathan/toolchain-cicd/saveLogsToArtifactsDir
```
Comment (alexeykazakov) [inline make/run-cicd-script.mk:2]: ?

Comment (rajivnathan) [reply make/run-cicd-script.mk:2]: this is what Matous was suggesting to do to test changes from our toolchain-cicd repo. This PR will not be merged.

**Comment by rajivnathan** [comment]
closed in favour of https://github.com/codeready-toolchain/toolchain-e2e/pull/516

**Pull request closed without merging**